src/GuzzleAdapter.php

```diff
@@ -59,7 +59,7 @@ private function request(string $method, Request $request): Response
         } catch (Exception $exception) {
             throw new NchanException(
                 $exception->getMessage(),
-                $exception->getCode(),
+                (int)$exception->getCode(),
```

Comment:
Change due to `phpstan` complaints as `Exception::__construct()` accepts an `int`, but `Exception::getCode()` return `mixed|int`. Explicit casting still leads to an error (which is configured to be ignored), but feels cleaner.